<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget_Monitor">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>647</width>
      <height>438</height>
     </rect>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>New Row</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>New Column</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>500</y>
      <width>759</width>
      <height>249</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>TX message</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_InputData_TX">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableWidget_TX">
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <row>
         <property name="text">
          <string>0</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>1</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>2</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>3</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>4</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Tx</string>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ID(hex)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Description</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>RTR</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D0</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D3</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D4</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D5</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D6</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>D7</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Count</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Time(ms)</string>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string/>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
        <item row="4" column="3">
         <property name="text">
          <string/>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Info message</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <widget class="QTableWidget" name="tableWidget_Info">
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>New Row</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>New Column</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>926</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
